pages/community/index.js

```diff
@@ -219,303 +108,288 @@ function CommunityIndexPage() {
-            {quickLinks.map((link) => (
-              <li
-                key={link.title}
-                className={`h-44 w-54 md:h-72 md:w-72 justify-between border border-primary-100 shadow-xl hover:shadow-lg transition-all duration-300 ease-in-out rounded-xl px-3 pb-5 bg-${link.bg} text-left flex justify-end items-end`}
-              >
-                <h1
-                  className={`text:lg md:text-3xl font-black text-${link.color}`}
-                >
-                  {link.title}
-                </h1>
-              </li>
-            ))}
-          </ul>
-        </div>
-        <div className="z-40 mt-28 bg-white w-full h-full-screen md:h-130 rounded-lg shadow-xl md:flex md:justify-between">
-          <div className="p-10 h-half-screen md:flex justify-between md:w-2/5 md:h-full flex-col text-center md:text-left">
-            <div>
-              <Heading
-                level="h2"
-                typeStyle="heading-md"
-                textColor="text-gray-200"
-              >
-                All community info, tracked
-              </Heading>
-            </div>
-            <div>
-              <Heading
-                level="h2"
-                typeStyle="heading-lg"
-                className="mt-10 md:mt-0"
-              >
-                Community Platform
-              </Heading>
-              <Heading
-                level="h2"
-                typeStyle="body-lg"
-                textColor="text-gray-700"
-                className="text-slate-500 text-sm mt-10"
-              >
-                AsyncAPI’s incredible community of developers, designers,
-                technical writers, hail from over 83 countries, actively
-                contributing, collaborating and mentoring others to build with
-                AsyncAPI.
-              </Heading>
-              <div className="mt-10"></div>
+      </div>
+      <div className="z-40 mt-10 bg-white w-full md:h-130 rounded-lg shadow-xl md:flex md:justify-between">
+        <div className="p-10 flex justify-between w-full md:w-2/5 h-auto flex-col text-center md:text-left">
+          <div>
+            <Heading
+              level="h2"
+              typeStyle="heading-md"
+              textColor="text-purple-300"
+            >
+              Support your rockstars
+            </Heading>
+          </div>
+          <div>
+            <Heading level="h2" typeStyle="heading-lg" className="mt-10">
+              Ambassador Programs
+            </Heading>
+            <Heading
+              level="h2"
+              typeStyle="body-lg"
+              textColor="text-gray-700"
+              className="text-slate-500 text-sm mt-10"
+            >
+              Launch community advocate programs that your members are proud to
+              be a part of. Powerful ambassador engagement analytics make it
+              easy to track and reward you contributors and build thriving
+              communities.
```

Comment:
```suggestion
              Launch OSS community programs that your community is proud to
              participate in. Let's build thriving OSS communities together!
```

Hmm, the phrase `"Powerful ambassador engagement analytics make it easy to track and reward you contributors"` sounds strange here. What **ambassador engagement analytics** do we have? This sounds made up or gimmicky.